Snakefile

```diff
@@ -27,13 +28,15 @@ hac_params = _config.config.hac_params
 FRAMEWORK = _config.config.container_framework
 print(f"Running {FRAMEWORK} containers")
 
-# Return the dataset dictionary from the config file given the label
-def get_dataset(_datasets, label):
+# Return the dataset or goldstandard dictionary from the config file given the label
+def get_dict(_datasets, label):
```

Comment:
It makes sense to reuse the same function for both datasets and gold standards. However, this function name is quite broad and should still have something to do with data.